File: src/main/java/com/realyoungk/sdi/dto/TelegramUpdateDto.java
package com.realyoungk.sdi.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public record TelegramUpdateDto(
        @JsonProperty("update_id") Long updateId,
        @JsonProperty("message") TelegramMessageDto message
) {
}

File: src/main/kotlin/com/realyoungk/sdi/dto/TelegramUpdateDto.kt
package com.realyoungk.sdi.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import com.fasterxml.jackson.annotation.JsonProperty

@JsonIgnoreProperties(ignoreUnknown = true)
data class TelegramUpdateDto(
    @JsonProperty("update_id") val updateId: Long,
    @JsonProperty("message") val message: TelegramMessageDto
)